src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -342,10 +376,10 @@ public function setTheme($theme)
             'bullist', 'numlist', 'outdent', 'indent',
         ],
         2 => [
-            'formatselect', '|',
-            'paste', 'pastetext', '|',
+            'blocks', '|',
+            'pastetext', '|',
```

Comment:
`formatselect` is now `blocks`: https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/#formatting-user-interface-name-changes

The paste button isn't supported by modern browsers.